Dockerfile

```diff
@@ -21,10 +21,11 @@ RUN go install -v ./...
 FROM docker.io/alpine:3
 WORKDIR /app
 
-RUN apk --no-cache add ca-certificates && mkdir /.cache && chmod -R g=u /.cache
+RUN mkdir /.cache && chmod -R g=u /.cache
+RUN apk --no-cache add ca-certificates
```

Comment:
Why 2 RUN?

Reply:
The `mkdir /.cache && chmod -R g=u /.cache` is completely independent of `apk --no-cache add ca-certificates`. Therefore I split it into two `RUN` statements.